runtime/src/snapshot_bank_utils.rs

```diff
@@ -1056,8 +1054,10 @@ pub fn bank_to_full_snapshot_archive(
         snapshot_storages,
         archive_format,
         snapshot_version,
-        maximum_full_snapshot_archives_to_retain,
-        maximum_incremental_snapshot_archives_to_retain,
+        // Since bank_to_snapshot_archive() is not called as part of normal validator operation,
```

Comment:
Was going to suggest putting `dev-context-only-utils` label on this function, but it sounds like wen-restart needs this function too so maybe not an option then ?